src/coverlet.msbuild.tasks/InstrumentationTask.cs

```diff
@@ -106,7 +106,7 @@ public override bool Execute()
                 var excludedSourceFiles = _excludeByFile?.Split(',');
                 var excludeAttributes = _excludeByAttribute?.Split(',');
 
-                Coverage coverage = new Coverage(_path, includeFilters, includeDirectories, excludeFilters, excludedSourceFiles, excludeAttributes, _includeTestAssembly, _singleHit, _mergeWith, _useSourceLink, _logger, (IInstrumentationHelper)DependencyInjection.Current.GetService(typeof(IInstrumentationHelper)));
+                Coverage coverage = new Coverage(_path, includeFilters, includeDirectories, excludeFilters, excludedSourceFiles, excludeAttributes, _includeTestAssembly, _singleHit, _mergeWith, _useSourceLink, _logger, (IFileSystem)DependencyInjection.Current.GetService(typeof(IFileSystem)), (IInstrumentationHelper)DependencyInjection.Current.GetService(typeof(IInstrumentationHelper)));
```

Comment:
Can you wrap initialization for best readability(I hope 😄 )?
```cs
Coverage coverage = new Coverage(_path, 
                                 includeFilters, 
                                 includeDirectorie
```

Reply:
Done